src/fidesops/service/connectors/saas_connector.py

```diff
@@ -87,6 +88,7 @@ def send(
         """
         try:
             prepared_request = self.get_authenticated_request(request_params)
+            log_request_for_debugging(prepared_request)  # Dev mode only
             response = self.session.send(prepared_request)
         except Exception:
```

Comment:
In addition, can we log the full `Exception` message in dev mode?

Reply:
Updated to log exception -

Reply:
Updated to log exception -